pinot-common/src/test/java/org/apache/pinot/common/utils/config/TableConfigSerDeTest.java

```diff
@@ -367,4 +381,12 @@ private void checkFieldConfig(TableConfig tableConfig) {
     assertNull(secondFieldConfig.getIndexType());
     assertNull(secondFieldConfig.getProperties());
   }
+
+  private void checkTableConfigWithUpsertConfig(TableConfig tableConfig, TableConfig tableConfigToCompare) {
```

Comment:
Can you make this similar to other config check? Pass in a  table config and check the actual value inside the config. Then serialize and deserialize (both string and ZNRecord) and check again